libpromises/files_copy.c

```diff
@@ -124,7 +124,7 @@ bool CopyRegularFileDisk(const char *source, const char *destination)
     unlink(destination);
 
     int dd = safe_open_create_perms(destination,
-                                    O_WRONLY | O_CREAT | O_TRUNC | O_EXCL | O_BINARY,
+                                    O_WRONLY | O_CREAT | O_EXCL | O_BINARY,
```

Comment:
This looks like an arbitrary/unrelated change.

Reply:
Well, it's just cleanup. `O_TRUNC` does nothing together with `O_CREAT` and `O_EXCL`.